bazel/repositories.bzl

```diff
@@ -44,6 +44,19 @@ def proxy_wasm_cpp_host_repositories():
         url = "https://github.com/bytecodealliance/wasmtime/archive/v0.26.0.tar.gz",
     )
 
+    http_archive(
+        name = "wamr",
+        build_file = "@proxy_wasm_cpp_host//bazel/external:wamr.BUILD",
+        sha256 = "f819b9ec866a12086233e578044dd8297e6be86f0f17807b16124f78a3652d4f",
+        url = "https://github.com/lum1n0us/wasm-micro-runtime/releases/download/WAMR-01-29-2021/source.zip",
+    )
+
+    # native.new_local_repository(
+    #     name = "wamr",
+    #     build_file = "@proxy_wasm_cpp_host//bazel/external:wamr.BUILD",
+    #     path = "/source/wamr",
+    # )
+
```

Comment:
Please remove the commented out section. If you're using it for local development, you can use Bazel's `--override_repository=wamr=/source/wamr` instead.

Reply:
seems can not assign a new *build_file* while using `--override_repository`. 
but, right, we should remove that debug information.